README.md

```diff
@@ -46,6 +47,24 @@ to include the theme located in this repository at `./kitty/edelweiss.conf`.
 echo 'include ~/.config/edelweiss/kitty/edelweiss.conf' >> ~/.config/kitty/kitty.conf
 ```
 
+### terminator
+
+Add a new profile to `.config/terminator/config` or change the `background_color`, `foreground_color`
+and `palette` in your default profile.
+
+```
+[profiles]
+  [[edelweiss]]
+    background_color = "#ffffff"
+    foreground_color = "#333333"
+    show_titlebar = False
+    scrollback_lines = 10000
```

Comment:
I suggest to either remove the unrelated config fields or provide a config file. Are the includes in terminator for themes?

Reply:
well :sweat_smile: it's a bit 'interesting' there is no thing like a theme or something you can install it's all part of a single config file this one -> 

```
$ cat .config/terminator/config 
[global_config]
  inactive_color_offset = 0.809989142237
[keybindings]
[profiles]
  [[default]]
    background_color = "#ffffff"
    foreground_color = "#333333"
    show_titlebar = False
    scrollback_lines = 10000
    palette = "#000000:#dc2626:#16a34a:#ca8a04:#2563eb:#db2777:#0891b2:#dddddd:#767676:#f87171:#4ade80:#facc15:#60a5fa:#f472b6:#22d3ee:#ffffff"
    login_shell = True
    bold_is_bright = True
[layouts]
  [[default]]
    [[[child1]]]
      parent = window0
      type = Terminal
    [[[window0]]]
      parent = ""
      type = Window
[plugins]
```

but im down to remove `show_titlebar` , `scrollback_lines` and `login_shell` as they are not related to the color config if you want me to